sgv2-restapi/src/main/resources/config.yaml

```diff
@@ -3,9 +3,9 @@ server:
   applicationContextPath: /
   connector:
     type: http
-    # 11-Feb-2022, tatu: For Stargate V1, REST+Docs were both on port 8082, but for Stargate V2
-    #    will separate the two and the "new" REST API inherits 8082
-    port: 8082
+    # 11-Feb-2022, tatu: This implementation will be removed soon, but until then
+    #    has to expose port OTHER than 8082 for ITs to work
+    port: 8083
```

Comment:
Why is this change needed? What integration tests would not work? I think this is not needed imo..